Library/Homebrew/Gemfile

```diff
@@ -0,0 +1,20 @@
+source "https://rubygems.org"
+
+require_relative "constants"
+
+group :test do
+  gem "parallel_tests"
+  gem "rspec"
+  gem "rspec-its", require: false
+  gem "rspec-wait", require: false
+end
+
+group :test, :coverage do
+  gem "codecov", require: false
+  gem "simplecov", require: false
+end
+
+group :rubocop do
+  gem "rubocop", HOMEBREW_RUBOCOP_VERSION
```

Comment:
Is there any reason to have these constants in the `constants` file rather than just having them in the `Gemfile` after this change?

Reply:
No, just wanted to keep the changes here as minimal as possible for the time being.

Reply:
Let's remove them then, thanks.